src/rules/__tests__/expect-expect.test.ts

```diff
@@ -151,6 +165,26 @@ ruleTester.run('expect-expect', rule, {
         },
       ],
     },
+    {
+      code: dedent`
+        const assertFoo = (input) => {
+          var valid = input === "foo"
+        }
+        it("should pass", () => assertFoo("foo"))
+      `,
+      options: [
+        {
+          assertFunctionNames: ['assert*'],
```

Comment:
We should be able to have a test case without this option being provided, to keep things simple (having _both_ test cases is good too).

e.g. like this https://github.com/jest-community/eslint-plugin-jest/pull/994/files#diff-be15fe30b6db731ff0d3ddcc2746228fd05cad63180b95c497b00869c2e7e601R144-R153